docs/developers/customize-karmada-scheduler.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Customize the scheduler
+---
+
+Karmada ships with a default scheduler that is described [here](../reference/components/karmada-scheduler.md). If the default scheduler does not suit your needs you can implement your own scheduler.
+Karmada's `Scheduler Framework` is similar to Kubernetes, but unlike K8s, Karmada needs to deploy applications to a group of clusters instead of nodes. According to the placement field of the user's scheduling policy and the internal scheduling plug-in algorithm, the user's application will be deployed to the desired cluster group.
+
+The scheduling process can be divided into the following four steps:
+* Predicate: filter inappropriate clusters
+* Priority: score the cluster
+* SelectClusters: select cluster groups based on cluster scores and `SpreadConstraint`
+* ReplicaScheduling: deploy the application replicas on the selected cluster group according to the configured replica scheduling policy
+
+ ![schedule process](../resources/developers/schedule-process.png)
+
+Among them, the plug-ins for filtering and scoring can be customized and configured based on the scheduler framework.
+
+The default scheduler has four in-tree plugins:
```

Comment:
four? There are 5 on the list.
```suggestion
The default scheduler has several in-tree plugins:
```
We don't need to count the plugins, because we probably add new plugins in the future.